api/config/env/env.json

```diff
@@ -0,0 +1,10 @@
+{
+ "NODE_ENV": "development",
+ "MEETUP_CLIENT_ID": "abo95nlv5evcaum5th922vlpia",
```

Comment:
@kushal2011 @mistryhardik @Chintan1193 @prachidurge @GayatriDarji @dhruvil2111 
Can we find a better way to maintain API keys off the public repo?

Reply:
@SnehPandya18 You can exclude these values and update the PR. We do not need these values to be pushed into the source control.